styles/index.css

```diff
@@ -128,9 +129,36 @@ body {
   border-radius: 7px;
   font-size: 20px;
   font-family: sans-serif;
+  background-color: thistle;
 }
 .skills {
   display: flex;
   margin: 20px;
   font-size: 25px;
 }
+.home {
+  color: white;
```

Comment:
Maybe a utility class like `c-w {color: white;}` would be better for the next classes which only have color: white;  ?

Reply:
I agree with Paul regarding the utility class. Maybe call it `text-white`?